src/app/globals.css

```
@@ -162,4 +162,136 @@
+    opacity: 1;
+    transform: scaleY(1.1);
+  }
+}
+
+/* Evidence timeline dot glow on card hover */
+@keyframes travel-timeline-glow {
+  0%, 100% {
+    filter: none;
+  }
+  50% {
+    filter: drop-shadow(0 0 3px currentColor);
```

Stylelint (17.7.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)